platform/src/test/java/platformstepdefinition/PF_Withdrawal.java

```diff
@@ -0,0 +1,56 @@
+package platformstepdefinition;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import base.NewBaseClass;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.junit.Cucumber;
+import platform.pageobjects.Authentication.LoginPage;
+import platform.pageobjects.Footer.FooterPage;
+import platform.pageobjects.Withdrawal.Withdrawal;
+
+/**
+ * @Author : Sahaj Balgunde
+ * @Date of Creation : 23 July 2021
+ * @Description: Additional Locator variables and Methods for Withdrawal.
+ */
+@RunWith(Cucumber.class)
+public class PF_Withdrawal extends NewBaseClass {
+
+	public WebDriver driver;
+
+	@Given("^Open Browser and URL (.+)$")
+	public void open_Browser_and_URL(String browser) throws Throwable {
+		driver = openbrowser(browser);
+	}
+
+	@Then("^User is on Withdrawal Page ticks the checkbox and withdraw account$")
+	public void user_is_on_Withdrawal_Page_ticks_the_checkbox_and_withdraw_account(String browser) throws Throwable {
+		LoginPage loginToPlatform = new LoginPage(driver);
+		loginToPlatform.loginToPlatform();
+		FooterPage footerobj = new FooterPage(driver);
+		footerobj.AccountFooter();
+		Withdrawal withdrawal = new Withdrawal(driver);
+		withdrawal.accountWithdrawal();
+
+		System.out.println("Account has been deleted.");
+
+	}
+
+	@Then("^Account gets deleted$")
+	public void Account_gets_deleted(String browser) throws Throwable {
+
+		System.out.println("Account has been deleted.");
```

Comment:
How are you validating account deletion?

Reply:
@csonparote On 2nd Then statement, account gets withdrawn. Need help for 3rd Then Statement.

Reply:
Try to login with old credentials
If error message then account deletion is validated

Reply:
Updated as requested.